src/screens/Login/LoginForm.tsx

```diff
@@ -100,14 +107,19 @@ export const LoginForm = ({
           service: serviceUrl,
           identifier: fullIdent,
           password,
+          authFactorToken,
```

Comment:
Maybe trim spaces? For easier copy paste.

Reply:
Good catch, done